src/mame/taito/taitopjc.cpp

```diff
@@ -1,13 +1,19 @@
 // license:BSD-3-Clause
-// copyright-holders:Philip Bennett
+// copyright-holders:Philip Bennett, Ville Linde
 /*
```

Comment:
You have no ground about changing copyright holder line. Are you Ville Linde?

Reply:
Removing this change then.